src/include/iscsi-client/dialogs.rb

```diff
@@ -466,7 +463,7 @@ def TargetsDialog
       w = CWM.CreateWidgets(["targets_table"], @widgets)
       contents = VBox(
         HBox(HSpacing(1), VBox(Ops.get_term(w, [0, "widget"]) { VSpacing(1) }), HSpacing(
-          1
+                                                                                  1
```

Comment:
NP: I think it would be better to put each `HBox` argument on a separate line to avoid this ugly indentation.